text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation on the Polkadot Asset Hub presents a significant financial barrier for many artists. By lowering the deposit requirements, we aim to encourage more artists to participate in the Polkadot NFT ecosystem, thereby enriching the diversity and vibrancy of the community and its offerings.
+
+## Stakeholders
+
+- **Artists**: Primary beneficiaries of the proposed change, particularly those who found the current deposit requirements prohibitive.
+- **KodaDot**: As the facilitator of artists' relations, KodaDot has a vested interest in making the platform more accessible.
+- **Polkadot Community**: Stands to benefit from an influx of artists, creators and diverse NFT collections, enhancing the overall ecosystem.
+
+Previous discussions have been held within the KodaDot community, as well as with artists expressing their concerns about the deposit amounts. Referencing to [Polkadot Forum conversation](https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262).
+
+## Explanation
+This RFC suggests modifying the smart contract governing collection creation on the Polkadot Asset Hub to require a lower deposit. The exact amount of the reduced deposit is yet to be determined and should be discussed and agreed upon by the stakeholders. The implementation of this change requires careful consideration of the network's integrity and the prevention of spam, possibly through alternative means such as rate limiting or account verification.
+
+**Prices and Proposed Prices on Polkadot Asset Hub:**
+_Scroll right_
+```
+| **Name**                  | **Current price implementation** | **Price if DOT = 5$**  | **Price if DOT goes to 50$**  | **Proposed Price in DOT** | **Proposed Price if DOT = 5$**   | **Proposed Price if DOT goes to 50$**|
+|---------------------------|----------------------------------|------------------------|-------------------------------|---------------------------|----------------------------------|--------------------------------------|
+| collectionDeposit         | 10 DOT                           | 50 $                   | 500 $                         | 0.1 DOT                   | 0.5 $                            | 5$                                   |
+| itemDeposit               | 0.01 DOT                         | 0.05 $                 | 0.5 $                         | 0.001 DOT                 | 0.005 $                          | 0.05$                                |
+| metadataDepositBase       | 0.20129 DOT                      | 1.00645 $              | 10.0645 $                     | 0.0020129 DOT             | 0.0100645 $                      | 0.100645$                            |
+| attributeDepositBase      | 0.2 DOT                          | 1 $                    | 10 $                          | 0.002 DOT                 | 0.01 $                           | 0.1$                                 |
```

Comment:
It would be nice to see these values in relation to what is actually being stored on-chain.  
Then we can compare these values to proven spam-resistant storage prices (like in the preimage pallet).  
It should be possible to calculate the DOT/Byte to make it comparable.  

Otherwise they are just as random as at genesis, now just 1/10 😉